index.js

```diff
@@ -156,8 +156,17 @@ module.exports = {
   // TODO check config
   buildElmAssets: (config, callback) => {
     validateConfig(config);
-    collectAssets(config, (err, assets) =>
-      writeElmFile(config, assets, callback));
+    collectAssets(config, (err, assets) => {
+      var sortedAssets =
```

Comment:
💭 I wonder if `collectAssets` can/should take on the responsibility of sorting assets. I like that `buildElmAssets` was a sequence of calls to exported functions.